package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shoutem/ui",
-  "version": "5.3.0",
+  "version": "5.3.1",
```

Comment:
With the other changes in Develop, this is a minor version, `5.4.0`, because we added `responsiveWidth` and `responsiveHeight` helpers, as well as added deprecation warning for `dimensionRelativeToIphone` which is to be removed with 6.0.0 eventually, being replaced with `responsiveWidth`.